src/events/acceleration.js

```diff
@@ -203,6 +244,18 @@ p5.prototype._handleMotion = function() {
     }
     old_max_axis = new_max_axis;
   }
+  var deviceShaken = this.deviceShaken || window.deviceShaken;
+  if (typeof deviceShaken === 'function') {
+    var accelerationChange = {};
```

Comment:
Sorry, I meant instead of this line `var accelerationChange = {};`, just create three separate variables:

```
var accelerationChangeX;
var accelerationChangeY;
var accelerationChangeZ;
```